benchexec/tools/metaval.py

```diff
@@ -41,24 +38,33 @@ class Tool(benchexec.tools.template.BaseTool2):
         "ultimateautomizer": "UAutomizer-linux",
     }
     PATH_TO_TOOL_MAP = {v: k for k, v in TOOL_TO_PATH_MAP.items()}
-    REQUIRED_PATHS = list(TOOL_TO_PATH_MAP.values()) + [
+
+    REQUIRED_PATHS = [
         "VERSION.txt",
         "metaval.py",
-        "metaval.sh",
     ]
 
     def __init__(self):
         self.lock = threading.Lock()
         self.wrappedTools = {}
 
-    def executable(self, toolLocator):
-        return toolLocator.find_executable("metaval.sh")
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("metaval.sh")
 
-    def name(self):
-        return "MetaVal"
+    def program_files(self, executable):
+        return (
+            [executable]
+            + BaseTool2._program_files_from_executable(
+                executable, self.REQUIRED_PATHS, parent_dir=True
+            )
+            + [
+                self._resource(executable, path)
+                for path in self.PATH_TO_TOOL_MAP.values()
+            ]
```

Comment:
Why use `_resource()` for this when the previous code (implicitly) used `_program_files_from_executable()` for these paths? I would just re-add the tool files to `REQUIRED_PATHS`.